plugins/obs-outputs/rtmp-stream.c

```diff
@@ -1064,8 +1085,11 @@ static bool init_connect(struct rtmp_stream *stream)
 	stream->dbr_orig_bitrate = (long)obs_data_get_int(vsettings, "bitrate");
 	stream->dbr_cur_bitrate = stream->dbr_orig_bitrate;
 	stream->dbr_est_bitrate = 0;
-	stream->dbr_inc_bitrate = stream->dbr_orig_bitrate / 10;
+	stream->dbr_inc_bitrate = stream->dbr_orig_bitrate / 5;
```

Comment:
What does `5` represent it? I would suggest putting this in a const variable with a meaningful name.

Reply:
Created a define called DBR_INC_RATE for that value

Reply:
Nice